src/applications/ivc-champva/10-7959a/config/submitTransformer.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable camelcase */
+import { transformForSubmit as formsSystemTransformForSubmit } from 'platform/forms-system/src/js/helpers';
+
+function getPrimaryContact(data) {
+  // For callback API we need to know what data in the form should be
+  // treated as the primary contact.
+  return {
+    name: data?.certifierName ?? data?.applicantName ?? false,
+    email: false, // We don't collect email
+    phone: data?.certifierPhone ?? data?.applicantPhone ?? false,
+  };
+}
```

Comment:
We need this transformation in every IVC form so we know who to contact. Added here